### PR TITLE
Feature/login redesign

### PR DESCRIPTION
**Pull request by leachtucker**
# Description
Redesign of the existing login/landing page.

Fixes #

- What work was done?
Removed unnecessary information. Restyled login form.
- Why was this work done?
To improve user experience upon logging in.
- What feature / user story is it for?
_As a parent, I would like a login page that is designed cohesively with the rest of the application._
- Any relevant links or images / screenshots
SC 1

## Type of change
- [X] New feature (non-breaking change which adds functionality)

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: src/__tests__/ChildReducer.test.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ describe('childReducer test suite', () => {
     losses: null,
     achievements: null,
     Ballots: [],
+    Streaks: 'test',
```
Comment (leachtucker) [inline src/__tests__/ChildReducer.test.js]: Team B's recent merge caused the ChildReducer test to fail. The initial state in the test file was missing the 'Streaks' property.

**Comment by leachtucker** [comment]
[As a parent, I would like a login page that is designed cohesively with the rest of the application.](https://trello.com/c/YMlQPD0s/50-as-a-parent-i-would-like-a-login-page-that-is-designed-cohesively-with-the-rest-of-the-application)

File: src/styles/less/global.less [hunk 2]
```diff
@@ -1 +1 @@
-@import url('https://fonts.googleapis.com/css2?family=Bangers&family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bangers&family=Nunito&family=Mulish&family=Open+Sans:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap');
```
Comment (leachtucker) [inline src/styles/less/global.less:1]: According to the Figma doc, we'll need these fonts for the buttons and headings of our app.
Mulish for buttons!

**Review by charlene-johnson — APPROVED** [review]
Everything looks great! I like the redesign

File: src/__tests__/ChildReducer.test.js (hunk 1 above)
Comment (Robert-Lark) [reply src/__tests__/ChildReducer.test.js]: Nice catch

**Review by Robert-Lark — APPROVED** [review]
Nice job - it's like looking at the prototype :)

**Review by cearc-sara — APPROVED** [review]
great job cleaning up the log in page looks much better this way!

**Pull request merged**